engine/src/main/java/org/hibernate/search/cfg/NumericFieldMapping.java

```diff
@@ -14,10 +14,11 @@
  */
 public class NumericFieldMapping extends FieldMapping {
 
-	private final Map<String, Object> numericField = new HashMap<String, Object>();
+	private final Map<String, Object> numericField = new HashMap<String, Object>( 2 );
 
-	public NumericFieldMapping(PropertyDescriptor property, EntityDescriptor entity, SearchMapping mapping) {
+	public NumericFieldMapping(String fieldName, PropertyDescriptor property, EntityDescriptor entity, SearchMapping mapping) {
```

Comment:
Technically it's an API change, but I'd say it's ok as it's not supposed to be called by clients really?

It'd be much better if there was a proper public API / impl split for the DSL as it's the case in the HV DSL.